theme/_login-theme-custom.scss

```diff
@@ -0,0 +1,59 @@
+#login_submit {
+  width: 100%;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.login-submit-btn-div {
+  position: relative;
+}
+
+.password_error {
+  border: 2px solid red;
+}
+.password_message {
+  display: none;
+}
+.password_message_error {
+  color: red;
+  display: block;
+}
+.password_message_success {
+  color: #80d48d;
+  display: block;
+}
+.password_success {
+  display: none;
+}
+
+.password-label-success {
+  color: #80d48d;
+}
+.password-label-error {
+  color: red;
+}
+.success {
+  border: 2px solid #80d48d;
+}
+
+.email_populated {
+  border: 2px solid blue;
+}
+
+.email_label_valid {
+  color: blue;
+}
+
+.not-submitable {
```

Comment:
Defining submitable/not-submitable classes is unnecessary.

Simply setting the disabled attribute on the button will both implement the desired interaction behavior, and inherit the default styling for a disabled button.